include/RAJA/exec-cuda/reduce_cuda.hxx

```diff
@@ -308,58 +333,44 @@ template <size_t BLOCK_SIZE, typename T>
 class ReduceMin< cuda_reduce<BLOCK_SIZE>, T > 
 {
 public:
```

Comment:
I am super-confused.... isn't this the third time I have approved these implementations of ReduceMin and ReduceMax?  Why do they keep showing up? @davidbeckingsale I'd like to get a handle on why this has shown up in three separate commits.